src/Protocol/Diaspora.php

```diff
@@ -621,106 +628,106 @@ private static function validPosting(array $msg)
 		$type = $element->getName();
 		$orig_type = $type;
 
-		Logger::debug("Got message type ".$type.": ".$msg["message"]);
+		Logger::debug("Got message type " . $type . ": " . $msg['message']);
```

Comment:
Same as above with other message, maybe use `DI::logger->foo()` instead.

Reply:
But since this is very unknown to me (I personally would go with a `sprintf('Some message with %s some %d variables.')` to have log messages as a complete string.